docs/UserGuide.md

```diff
@@ -22,15 +22,16 @@
 ## Getting Started
 
 First, ensure that you have the necessary prerequisites:
-* **JDK `1.8.0_60`** or later
-* **git `2.14`** or later on the command line (run `git --version` in your OS terminal to confirm)
+* **Java 8** (JRE `1.8.0_60`) or later. You may download Java [here](https://www.java.com/en/).
+* **git `2.14`** or later on the command line (run `git --version` in your OS terminal to confirm). You may download git [here](https://git-scm.com/downloads).
 
 Next, download the latest executable Jar from our [releases](https://github.com/reposense/RepoSense/releases/latest).
 
 The simplest use case for RepoSense is to generate a report for the entire history of a repo. Here are the steps:
 1. Generate the report for the repo by executing the following command in a terminal:<br/>
    Format : `java -jar RepoSense.jar --repo FULL_REPO_URL` (note the `.git` at the end)<br>
    Example: `java -jar RepoSense.jar --repo https://github.com/reposense/RepoSense.git`
+   > Note: The above command will analyze the commits made within one month from the date of report generation. Append `--since d1` if you wish to analyze from the date of the first commit. 
 1. The previous step analyzes the default branch of the repo and creates the report in a directory named `reposense-report`. Run the following command to view the report (it will open up in your default Browser):<br/>
    `java -jar RepoSense.jar --view reposense-report`
```

Comment:
Within the `chart panel` section, please update the diagram and it's description to the latest version with the updated title format and view commits button.

Reply:
@jylee-git can u close the tab pane and screen shot the ramp chart so that it is more wider? The current diagram takes up quite some space in the UG.